dask_ml/datasets.py

```diff
@@ -64,12 +64,14 @@ def make_counts(
     rng = dask_ml.utils.check_random_state(random_state)
 
     X = rng.normal(0, 1, size=(n_samples, n_features), chunks=(chunks, n_features))
-    informative_idx = rng.choice(n_features, n_informative, chunks=n_informative)
+    informative_idx = rng.choice(
+        n_features, n_informative, chunks=n_informative, replace=False
```

Comment:
This is a good change.

I think this PR will get merge faster if all the changes in this file are left to another PR.

Reply:
I think it'd be good to do some better error handling in this function (like making sure that `n_informative < n_features`). I think a future PR would be a great place for that alongside workarounds for the items in https://github.com/dask/dask-ml/pull/914#discussion_r854348163 if they're still relevant.

Past the changes in this file, this PR LGTM!

Reply:
Awesome, thank you! I have reverted the two changes from the `datasets.py` file. When I did so, I had to make one small change to the `atol` condition of the tests to make it pass for the `PoissonRegressor`. The original condition checked to the `1e-4`, and the maximum with the dataset update was ~0.00014, so I updated the condition to restrict only to `2e-4`.

A future PR for the other changes to the `datasets.py` file sounds like a good idea to me. I would be happy to work on that at some point, but may not be able to do so in the short term.